routing/base/astar_algorithm.hpp

```diff
@@ -12,6 +12,13 @@
 namespace routing
 {
 
+template <typename TVertexType>
+struct RoutingResult
+{
+  vector<TVertexType> path;
```

Comment:
Ok for c_tor. But `path` and `distance` fields are fields of the RoutingResult structure. They have not  personal getter and setter functions. And they used directly. So we may not use `m_` prefix for this case.